src/kata7.getEmployerRole.js

```diff
@@ -1,3 +1,5 @@
-const getEmployerRole = (employeeName, employees) => {};
+const getEmployerRole = (employeeName, employees) => {
+  return employees.find(employee => employee.name == employeeName).role
```

Comment:
Very nice solution :) 

the only thing is to use === instead of == and it can be refactored like this as one line:

````
const getEmployerRole = (employeeName, employees) =>
  employees.find((employee) => employee.name === employeeName).role;
`````